inspector/loadavg.go

```diff
@@ -114,6 +149,10 @@ func NewLoadAvg(driver *driver.Driver, _ ...string) (Inspector, error) {
 			//      Command: `sysctl -n vm.loadavg | awk '{ printf "%.2f %.2f %.2f ", $2, $3, $4 }'`,
 			Command: `top -l 1 | grep "Load Avg:" | awk '{print $3, $4, $5}'`,
 		}
+	} else if details.IsWindows {
+		loadavg = &LoadAvgWin{
+			Command: `wmic cpu get loadpercentage`,
```

Comment:
This function is available for all win versions?

Reply:
Yeah